src/compoenents/CustomerMailDetailPannel/MapDashboard.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { Box, Button, Grid, Paper, Typography } from '@mui/material';
+import { ComposableMap, Geographies, Geography } from 'react-simple-maps';
+
+const geoUrl =
+  "https://raw.githubusercontent.com/zcreativelabs/react-simple-maps/master/topojson-maps/world-110m.json";
+
+const MapDashboard: React.FC = () => {
+  return (
+    <Box p={3}>
+      <Grid container spacing={3}>
+        {/* Sidebar and Weather Information */}
+        <Grid item xs={12} md={3}>
+          <Grid container spacing={3}>
+            {/* Sidebar */}
+            <Grid item xs={12}>
+              <Paper elevation={3} sx={{ p: 2 }}>
+                <Button fullWidth variant="contained" color="primary" sx={{ mb: 2 }}>
+                  Yield
+                </Button>
+                <Button fullWidth variant="contained" color="primary" sx={{ mb: 2 }}>
+                  Stress
+                </Button>
+                <Button fullWidth variant="contained" color="primary">
+                  Disease
+                </Button>
+              </Paper>
+            </Grid>
+            {/* Weather Information */}
+            <Grid item xs={12}>
+              <Paper elevation={3} sx={{ p: 2 }}>
+                <Typography variant="h6">Weather Information</Typography>
+                <Typography variant="body1">Mon, Jul 22</Typography>
+                <Typography variant="body2">Kotte</Typography>
+                <Typography variant="body2">Temperature: 31.7°C</Typography>
+                <Typography variant="body2">Humidity: 85%</Typography>
+              </Paper>
+            </Grid>
+          </Grid>
+        </Grid>
+
+        {/* Map */}
+        <Grid item xs={12} md={9}>
+          <Paper elevation={3} sx={{ p: 2, height: '100%' }}>
+            <ComposableMap>
+              <Geographies geography={geoUrl}>
+                {({ geographies }) =>
+                  geographies.map((geo) => (
```

Comment:
line number 47 u used geographies parameter and line number 48 u have passed geo parameter to the arrow func cant we explicitly define the type for the geo parameter and geographies parameter? i got some type errors on those parameters. didnt u?
```suggestion
                {({ geographies: ExpectedType1 }) =>
                  geographies.map((geo: ExpectedType2) => (
```